packages/teleport/src/components/StepSlider/StepSlider.story.tsx

```diff
@@ -0,0 +1,319 @@
+/**
+ * Copyright 2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import StepSlider, { SliderProps } from './StepSlider';
+import { Text, Card, ButtonPrimary, ButtonLink, Box } from 'design';
+
+export default {
+  title: 'Teleport/StepSlider',
+};
+
+const singleFlow = { default: [Body1, Body2] };
+export const SingleStaticFlow = () => {
+  return (
+    <Card bg="primary.light" my="5" mx="auto" width={464}>
+      <Text typography="h3" pt={5} textAlign="center" color="light">
+        Static Title
+      </Text>
+      <StepSlider<typeof singleFlow>
+        flows={singleFlow}
+        currFlow={'default'}
+        render={(slider, ref, parentStyle) => (
```

Comment:
The `render` method here doesn't feel very "reacty". Why would we do this instead of having the `StepSlider` return itself wrapped in a `Box` with the `ref` and `parentStyle` set?

Reply:
you're right, i don't remember why i did it this way :thinking: